LuckyDragon/app/src/main/java/com/example/luckydragon/ProfileActivity.java

```diff
@@ -41,24 +28,51 @@ enum Mode {
         ORGANIZER,
         ADMIN,
     }
-
     private User user;
+    private GlobalApp.ROLE role;
+    private ProfileView profileView;
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_profile);
         getSupportActionBar().setTitle("Profile");
 
-        // Unpack intent
-        Intent intent = getIntent();
+        // Get user from global app
         user = ((GlobalApp) getApplication()).getUser();
-        String role = intent.getStringExtra("role");
+        role = ((GlobalApp) getApplication()).getRole();
+
+        // Create profile view
+        profileView = new ProfileView(user, this);
+
+        // If user exists, update view
+        if(user != null) {
+            user.notifyObservers();
+        }
```

Comment:
user.notfiyObservers is called when you create ProfileView already. Follow the the Observer.startObserving method